ui/e2e/Dockerfile.sauceconnect

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu
```

Comment:
What about `java:openjdk-8-jdk-alpine` or one of the other alpine tags for the java image?